examples/log-api/log_api.bal

```diff
@@ -35,14 +19,34 @@ public function main() {
     log:printError("error log",
         id = isolated function() returns int { return 845315;}, name = "foo");
 
-    // Users can also pass a key/value pair where the value is an error stack trace.
-    f1();
+    // You can also pass a key/value pair in which the value is an error stack trace.
+    error err = f1();
+    log:printError("error log", stackTrace = err.stackTrace().callStack,
+     username = "Alex92", id = 845315);
 
     // Optionally, an error can be passed to the functions.
     error e = error("something went wrong!");
     log:printError("error log with cause", 'error = e, id = 845315,
         name = "foo");
 
+    // The log level can be configured via a Ballerina configuration file.
+    // To set the global log level, place the entry given below in the `Config.toml` file:
+    //
+    // ```
+    // [ballerina.log]
+    // level = "[LOG_LEVEL]"
+    // ```
+
+    // Each module can also be assigned its own log level. To assign a
+    // log level to a module, provide the following entry in the `Config.toml` file:
+    //
+    // ```
+    // [[ballerina.log.modules]]
+    // name = "[ORG_NAME]/[MODULE_NAME]"
+    // level = "[LOG_LEVEL]"
+    // ```
+
+    // By default, log messages are logged to the console in LogFmt format.
```

Comment:
Shall we add these comments to the description file

Reply:
Updated in 54cc3cb